mysql-test/suite/innodb/r/mem_pressure.result

```diff
@@ -0,0 +1,23 @@
+#
+# MDEV-24670 avoid OOM by linux kernel co-operative memory management
+#
+CREATE TABLE t1 (t TEXT) ENGINE=InnoDB;
+INSERT INTO t1 SELECT CONCAT(REPEAT('junk text', 500), CAST(seq AS CHAR)) FROM seq_1_to_1000;
+SELECT CAST(VARIABLE_VALUE AS INTEGER) INTO @dirty_prev FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME='Innodb_buffer_pool_pages_dirty';
+SELECT CAST(VARIABLE_VALUE AS INTEGER) INTO @flush_prev FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME='Innodb_buffer_pool_pages_flushed';
+set debug_dbug="d,trigger_garbage_collection";
+SET GLOBAL innodb_buffer_pool_size=@@innodb_buffer_pool_size;
```

Comment:
Why are we checking this? This could actually result in different values depending on the build parameters, because the `buf_block_t` will be allocated from the buffer pool as well.

Reply:
Oh, sorry, we are not displaying the size of `innodb_buffer_pool_size`, but using the dummy `SET GLOBAL` as a special trigger mechanism that is enabled by the `debug_dbug`.